tests/e2e/project/commits.spec.ts

```diff
@@ -138,12 +138,13 @@ test("pushing changes while viewing history", async ({ page, peerManager }) => {
     name: "alice",
     gitOptions: gitOptions["alice"],
   });
-  await alice.startHttpd(8090);
+  await alice.startHttpd();
+  const aliceHttpdUrl = `${alice.httpdBaseUrl.hostname}:${alice.httpdBaseUrl.port}`;
   await alice.startNode();
   const { rid, projectFolder } = await createProject(alice, "alice-project");
-  await page.goto(`/seeds/127.0.0.1:8090/${rid}`);
+  await page.goto(`/seeds/${aliceHttpdUrl}/${rid}`);
```

Comment:
Couldn’t you do the following
```suggestion
  await page.goto(`${alicel.uiUrl()}/${rid}`);
```

Reply:
Yeah that's a good one 👍

Reply:
Done ✅